src/YOLO/index_test.js

```diff
@@ -25,14 +25,15 @@ describe('YOLO', () => {
 
```

Comment:
The test error is:
```javascript
TypeError: Cannot read property 'filterBoxesThreshold' of undefined
```
can you try defining the variable here first. Like this: https://github.com/ml5js/ml5-library/blob/master/src/ImageClassifier/index_test.js#L20